web-wallet/src/lib/containers/IconHeaderToggleCard/IconHeaderToggleCard.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import { Card, Icon, Switch } from "$lib/dusk/components";
+
+  import "./IconHeaderToggleCard.css";
+
+  /** @type {string | undefined} */
+  export let iconPath = undefined;
+
+  /** @type {string} */
+  export let heading;
+
+  /** @type {CardGap} */
+  export let gap = "default";
+
+  /** @type {boolean} */
+  export let onSurface = false;
+
+  /** @type {boolean | undefined} */
+  export let isToggled = undefined;
```

Comment:
Since this is a boolean, it should only be possible to be `true` or `false`

Reply:
I have introduced a [IconHeadingCard.svelte](https://github.com/dusk-network/rusk/pull/1655#diff-7d32426cf3eb050148441d625c4918c273410ff32e39e34c14148ba4f1547e53) and [web-wallet/src/lib/containers/Cards/ToggleableCard.svelte.](https://github.com/dusk-network/rusk/pull/1655/files#diff-3acf5464cf9f3710d611106722a2d8ffa120a6538eb433f08ad6cc30e50e6991).

The ToggleableCard is the functionality of IconHeadingCard + the toggle.

This is to avoid potential ambiguity that could have been caused by having one component instead with the isToggled allowing a boolean or an undefined value.